index.bs

```diff
@@ -206,268 +177,455 @@ interface ImageCapture {
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>whiteBalanceMode</code></dfn></dt>
+  <dd>See <a>white balance mode</a> constrainable property.</dd>
 
-  <dt><dfn attribute for="MediaSettingsRange"><code>min</code></dfn></dt>
-  <dd>The minimum value of this setting</dd>
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>colorTemperature</code></dfn></dt>
+  <dd>See <a>color temperature</a> constrainable property.</dd>
 
-  <dt><dfn attribute for="MediaSettingsRange"><code>current</code></dfn></dt>
-  <dd>The current value of this setting</dd>
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>exposureMode</code></dfn></dt>
+  <dd>See <a>exposure</a> constrainable property.</dd>
 
-  <dt><dfn attribute for="MediaSettingsRange"><code>step</code></dfn></dt>
-  <dd>The minimum difference between consecutive values of this setting.</dd>
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>exposureCompensation</code></dfn></dt>
+  <dd>See <a>exposure compensation</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>iso</code></dfn></dt>
+  <dd>See <a>iso</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>focusMode</code></dfn></dt>
+  <dd>See <a>focus mode</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>pointsOfInterest</code></dfn></dt>
+  <dd>See <a>points of interest</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>brightness</code></dfn></dt>
+  <dd>See <a>brightness</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>contrast</code></dfn></dt>
+  <dd>See <a>contrast</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>saturation</code></dfn></dt>
+  <dd>See <a>saturation</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>sharpness</code></dfn></dt>
+  <dd>See <a>sharpness</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>zoom</code></dfn></dt>
+  <dd>See <a>zoom</a> constrainable property.</dd>
+
+  <dt><dfn dict-member for="MediaTrackConstraintSet"><code>torch</code></dfn></dt>
+  <dd>See <a>torch</a> constrainable property.</dd>
 </dl>
 
-# FillLightMode
+## {{MediaTrackSettings}} dictionary ## {#mediatracksettings-section}
+
+When the {{getSettings()}} method is invoked on a video stream track, the user agent must return the extended {{MediaTrackSettings}} dictionary, representing the current status of the underlying user agent.
 
 <pre class="idl">
-  enum FillLightMode {
-    "unavailable",
-    "auto",
-    "off",
-    "flash",
-    "torch"
+  partial dictionary MediaTrackSettings {
+    MeteringMode      whiteBalanceMode;
+    MeteringMode      exposureMode;
+    MeteringMode      focusMode;
+    sequence&lt;Point2D> pointsOfInterest;
```

Comment:
The > needs escaping.

Also, a sequence of Point2D would be an entirely new constraint. Constraining on it would need to be worked out. You'd probably need to define new Constrain* types for it. The closest to model on may be ConstrainDOMString, which is array-based.

I also confess I don't know what pointsOfInterest is for here in practice, or how it applies to a video stream. How is it intended to be used?